packages/mongodb-memory-server-core/src/util/MongoInstance.ts

```diff
@@ -16,11 +16,85 @@ const log = debug('MongoMS:MongoInstance');
 
 export type StorageEngine = 'devnull' | 'ephemeralForTest' | 'mmapv1' | 'wiredTiger';
 
+/**
+ * Overwrite replica member-specific configuration
+ *
+ * @see {@link https://docs.mongodb.com/manual/reference/replica-configuration/#replica-set-configuration-document-example}
+ *
+ * @example
+ * ```ts
+ * {
+ *   priority: 2,
+ *   buildIndexes: false,
+ *   votes: 2,
+ * }
+ * ```
+ */
+export interface ReplicaMemberConfig {
+  /**
+   * A boolean that identifies an arbiter.
+   * @defaultValue `false` - A value of `true` indicates that the member is an arbiter.
+   */
+  arbiterOnly?: boolean;
+
+  /**
+   * A boolean that indicates whether the mongod builds indexes on this member.
+   * You can only set this value when adding a member to a replica set.
+   * @defaultValue `true`
+   */
+  buildIndexes?: boolean;
+
+  /**
+   * The replica set hides this instance and does not include the member in the output of `db.hello()` or `hello`.
+   * @defaultValue `true`
+   */
+  hidden?: boolean;
+
+  /**
+   * A number that indicates the relative eligibility of a member to become a primary.
+   * Specify higher values to make a member more eligible to become primary, and lower values to make the member less eligible.
+   * @defaultValue 1.0 for primary/secondary; 0 for arbiters.
+   */
+  priority?: number;
+
+  /**
+   * A tags document contains user-defined tag field and value pairs for the replica set member.
+   * @defaultValue `null`
+   * @example
+   * ```ts
+   * { "<tag1>": "<string1>", "<tag2>": "<string2>",... }
+   * ```
+   */
+  tags?: any;
+
+  /**
+   * Mongodb 4.x only - The number of seconds "behind" the primary that this replica set member should "lag".
+   * @deprecated For mongodb 5.x, use `secondaryDelaySecs` instead.
```

Comment:
could this be removed? at least for as long as 4.0.x is the default for this package